features/fetching_articles_from_news_api.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Website should contain news from API
+  As a owner
+  In order to provide more articles for users to read
+  We would like to pull articles from other sources using an API
+
+
+Scenario: User visit the Index page
+  Given I am on the "Index" page
+  Then I should see 5 articles
```

Comment:
One thing with this test is that you're making an actual call to the real API during the test to retrieve articles. This will have an impact on the performance of this test. It'll be slow because of the network call but also if the API doesn't work then your tests will fail. 

One way to deal with this is to create stubs for network requests using a gem like [WebMock](https://github.com/bblimke/webmock). You can either deal with this now on this PR (if you chose to) or create a ticket to address this separately.

Reply:
Should this be a new PR to enhance the API feature or would you like us to rewrite this story/tasks/tests?

Reply:
You can merge if you've fixed the other issues I've mentioned above